README.md

```diff
@@ -16,6 +16,13 @@ You can contact us for support on our [chat server](https://discord.gg/V3vSCs7)
 ```js
 const fs = require('fs');
 const ytdl = require('ytdl-core');
+/**
+ * or for Babel / TypeScript:
+ * import ytdl from 'ytdl-core';
+ * Note: Requires --esmoduleinterop for TypeScript
+ * use "import * as ytdl" with --allowsyntheticdefaultimport
+ * or "import ytdl = require('ytdl-core')" with neither
```

Comment:
Maybe this line should go at the top since it uses no options? And the current first one format would look better with the format from the current 2nd. So something like

    import ytdl from 'ytdl-core'; with --esmoduleinterop
    import * as ytdl from 'ytdl-core'; with --allowsynthethicdefaultimport